### PR TITLE
Add test for all rates greater than session min rates.

### PR DESCRIPTION
**Pull request by sunash**
This test fails on some cases as of this commit. This PR serves to discuss how we'll fix the failing test.

### CODE REVIEW

File: acnportal/algorithms/preprocessing.py [hunk 1]
```diff
@@ -178,3 +147,29 @@ def apply_minimum_charging_rate(
...
+
+def remove_finished_sessions(active_sessions, infrastructure, period):
+    """ Remove any sessions where the remaining demand is less than threshold.
+
+    Args:
+        active_sessions (List[SessionInfo]): List of SessionInfo objects for
+            all active charging sessions.
+        infrastructure (InfrastructureInfo): Description of the charging
+            infrastructure.
+        period (int): Length of each time period in minutes.
```
Comment (sunash) [inline acnportal/algorithms/preprocessing.py:160]: In general, can't the period be a float?

**Review by sunash — COMMENTED** [review]
LGTM. I added some minor changes to docs and formatting.

File: acnportal/algorithms/preprocessing.py (hunk 1 above)
Comment (zach401) [reply acnportal/algorithms/preprocessing.py]: I have never tried it. I think things may break, but I'm not sure. I believe that we make an implicit assumption that it divides 60 evenly in some places (which could still work for a float). But I would need to go over the code and make sure.

**Comment by zach401** [comment]
LGTM.

**Pull request merged**